bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -158,6 +165,28 @@ class VideoProvider extends Component {
 
     this.visibility.onVisible(this.unpauseViewers);
     this.visibility.onHidden(this.pauseViewers);
+
+    this.currentWebcamsStatsInterval = setInterval(() => {
```

Comment:
Do you think we should be able to opt-out of re-running this every 5s on a device with good connection but low cpu? Or we assume that if there are webcams visible, then we should be able to re-run it